src/Rule.ts

```diff
@@ -7,13 +7,11 @@ import assert from "assert";
 export abstract class Rule {
 
     abstract readonly id: string;
-    readonly repair: boolean;
+    readonly repair = process.env.repair == 'true';
```

Comment:
<details>
<summary>:warning: <strong>This code uses `==`/ `!=` to perform an equality/inequality comparison.</strong> (click for details)</summary>

A possible improvement would be to use `===`/`!==` instead.  
  
There are several advantages of using `===`/`!==` even though the types of both the expressions being compared are the same as:  
**Consistency:** Using `==`/`!=` doesn't benefit in any way, so why deviate from the simple rule of avoiding it?

**Simplicity and performance:** In general, `=== `/`!==` is the simpler operation, because it doesn't convert its operands. The performance landscape across JavaScript engines is uneven, but on most browsers `===`/`!==` is as fast or faster than `==`/`!=`.

**[More details](https://confluence.devfactory.com/x/8FrfFg)**

## The improvement suggestion in this case
```ts  
readonly repair = process.env.repair === 'true';
```  =
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="560bb436-e530-4736-a9e4-58b12450b495" data-finding-id="383736560" href="none" codereview-uuid-reserved-tag data-business-id="6dc0ae0d-4d36-43a5-8caa-b9d612986ecd" data-analyzer="js" />)